packages/common/src/GRPCClient.ts

```diff
@@ -427,12 +427,12 @@ export default class ConcordiumNodeClient {
      *
      * @param transactionHash a transaction hash as a bytearray.
      * @param timeoutTime the number of milliseconds until the function throws error.
-     * @returns A blockhash as a byte array.
+     * @returns BlockItemSummary of the transaction.
      */
     async waitForTransactionFinalization(
         transactionHash: HexString,
         timeoutTime?: number
-    ): Promise<HexString> {
```

Comment:
I think we should return the entire `BlockItemSummaryInBlock`, as the block hash might be useful for people as well :)